rmk/src/keymap.rs

```diff
@@ -185,6 +187,29 @@ impl<'a, const ROW: usize, const COL: usize, const NUM_LAYER: usize, const NUM_E
         }
     }
 
+    /// Translate a KeyAction to ExtractedKeyAction
+    pub(crate) fn into_morse(
+        &self,
+        key_action: &KeyAction,
+        _pos: KeyboardEventPos,
+    ) -> Option<Morse<TAP_DANCE_MAX_TAP>> {
+        match key_action {
+            KeyAction::TapHold(tap_action, hold_action) => Some(
+                if let Action::Key(keycode) = tap_action
+                    && keycode.is_home_row()
+                    && let Action::Modifier(modifier) = hold_action
+                {
+                    Morse::new_hrm(*tap_action, *modifier)
+                } else {
+                    Morse::new_tap_hold(*tap_action, *hold_action)
```

Comment:
Do you think if it's better to change the default tap-hold mode to `HoldOnOtherKeyPress`?

Reply:
I agree.